pkg/exporter/sumologicexporter/exporter_test.go

```diff
@@ -541,11 +541,55 @@ func TestPushJSONLogs_AttributeTranslation(t *testing.T) {
 
 	testcases := []struct {
 		name       string
+		logs       pdata.Logs
 		configFunc func() *Config
 		callbacks  []func(w http.ResponseWriter, req *http.Request)
 	}{
+		{
+			name: "logs from different files",
```

Comment:
This sounds like an incomplete sentence to me :) Logs from different files ... ?

Reply:
Changed it to `logs from different files are sent with different metadata`